### PR TITLE
Update 5_Organizational_Agreements.md

### PR DESCRIPTION
**Pull request by ssteinbuss**
Adding current text for running data space instances

### CODE REVIEW

File: documentation/5_Organizational_Agreements.md [hunk 1]
```diff
@@ -65,4 +65,162 @@ process is defined by the IDSA certification working group.
...
+migrating/ developing existing data sharing initiatives, (intra data
+space interoperability), and embedding them in the European ambition of
+a federation of interoperable data spaces (inter data space
+interoperability).
+
+![Relationship of OpenDEI Building
+Blocks and data space instances](media/media/image18.png)
+
+This structure of a data space reflects the role of the IDS data space
+framework in relation to the broadly used data sharing agreement
```
Comment (PeterKoen-MSFT) [inline documentation/5_Organizational_Agreements.md:113]: which "broadly used data sharing agreement framework" is referenced here? I'm not aware of one that would be aligned with the IDSA Rulebook.

File: documentation/5_Organizational_Agreements.md [hunk 2]
```diff
@@ -65,4 +65,162 @@ process is defined by the IDSA certification working group.
...
+    expectations, responsibilities, and business processes.
+
+- *Legal level,* to make sure that organizations under different legal
+    jurisdictions and frameworks can share data with common legally
+    binding conditions.
+
+Governance on the identified topics distinguishes between two
+development lines for data space instances:
+
+- *Intra data space interoperability,* between the data space
```
Comment (PeterKoen-MSFT) [inline documentation/5_Organizational_Agreements.md:143]: missing participants as the core element of interoperability in intra & inter dataspace interop

File: documentation/5_Organizational_Agreements.md [hunk 3]
```diff
@@ -65,4 +65,162 @@ process is defined by the IDSA certification working group.
...
+The various governance instruments that may be considered by the IDSA
+are listed in the Table below. These governance tools each describe a
+different aspect of a certain activity and are used in the next section
+to provide a multidimensional overview of the intra and inter data
+spaces governance.
+
+| **Governance Instrument** | **Governance instrument description**      |
+|--------------------------------- |---------------------------------|
+| *Standardization* | Ensuring that the tasks, processes, and guidelines around this activity are formalized, documented, and  aligned between data spaces instances and the IDSA. The standardization efforts can generally be used as a  blueprint or starting point for the stakeholders.|
+| *Certification* | Validating that stakeholders act according to the standardized way of working. Certification is divided into component certification (certification of technical/software components) and organizational certification (organizational and legal processes).    |
```
Comment (PeterKoen-MSFT) [inline documentation/5_Organizational_Agreements.md:181]: It's not Certification, but rather Compliance (with standardized protocols). IDSA will ensure that Certification includes compliance with the protocols, but Certification covers a lot more than interoperability on a technical level.

File: documentation/5_Organizational_Agreements.md [hunk 4]
```diff
@@ -65,4 +65,162 @@ process is defined by the IDSA certification working group.
...
+| *Development* | The activity may require (software) development tasks as part of the realization, which should be compliant with the standardization activities and best followed by a certification. |
+| *Operations*           | The operations are about the exploitation and usage of the developed components. The operations can be certified as part of the organizational certification. |
```
Comment (PeterKoen-MSFT) [inline documentation/5_Organizational_Agreements.md:183]: Operations is not a part of interoperability. for two participants to be interoperable on a technical layer they both need to adhere to the protocols, how they manage their IT infrastructure is completely irrelevant for interoperability

File: documentation/5_Organizational_Agreements.md [hunk 5]
```diff
@@ -65,4 +65,162 @@ process is defined by the IDSA certification working group.
...
+| *Communication*        | The dissemination of the activity is an important aspect which might include the awareness of the standardization and certification, but also contains marketing aspects. |
```
Comment (PeterKoen-MSFT) [inline documentation/5_Organizational_Agreements.md:184]: again, irrelevant for interoperability on the technical layer. It's the adherence to standardized protocols, definitely not marketing....

File: documentation/5_Organizational_Agreements.md [hunk 6]
```diff
@@ -65,4 +65,162 @@ process is defined by the IDSA certification working group.
...
+| *Support*   | The support activities include the structured assistance from stakeholders involved in the operation, development, certification, and communication activities. |
```
Comment (PeterKoen-MSFT) [inline documentation/5_Organizational_Agreements.md:185]: support is not part of interoperability.

File: documentation/5_Organizational_Agreements.md [hunk 7]
```diff
@@ -65,4 +65,162 @@ process is defined by the IDSA certification working group.
...
+
+> update text
+
+There will be no single data space. Individual sectors or communities
+are expected to develop their own data space instances. Being able to
+seamlessly share data over these data space instances brings clear
+benefits. It extends the reach and scope of accessible data and allows
+the development of new business models and services across sectors and
+regions. Interoperability between data space instances adds major value,
+resulting in a federation of interoperable data spaces.
```
Comment (PeterKoen-MSFT) [inline documentation/5_Organizational_Agreements.md:197]: interoperability does not result in a federation of dataspaces. Federations can be created through common legal and business processes and agreements, but they do not exist automatically.

File: documentation/5_Organizational_Agreements.md [hunk 8]
```diff
@@ -65,4 +65,162 @@ process is defined by the IDSA certification working group.
...
+
+The federation of data spaces require a common set of reusable standards
```
Comment (PeterKoen-MSFT) [inline documentation/5_Organizational_Agreements.md:199]: Federations (plural) as it is extremely unlikely that there will be one federation of dataspaces to govern them all.

**Review by PeterKoen-MSFT — CHANGES_REQUESTED** [review]
Critical aspects on interoperability being based on participants interacting in multiple dataspaces is missing.
This text assumes that "European" dataspaces are the global norm and is missing the international aspect of interoperability.
Several factual errors in describing interoperability that need to be fixed before this can become a statement with normative character in the IDSA rulebook.

File: documentation/5_Organizational_Agreements.md [hunk 9]
```diff
@@ -65,4 +65,162 @@ process is defined by the IDSA certification working group.
...
+2. the ambition of a federation of interoperable data spaces.
+
+This implies that IDSA considers its development and deployment initiatives
+in the broader context of these two areas:
+
+The striving for interoperability within a data space (intra) so that
+IDS can provide a gradual migration path within a data space instance or
+data sharing initiative and preparing for interoperability between
+multiple data space instances or data sharing initiatives (inter) to
+pave the way for the federation of interoperable data spaces - as
```
[the remark this answers is not in the document]
Comment (PeterKoen-MSFT) [reply documentation/5_Organizational_Agreements.md:86]: Having thought about this a bit longer I need to change my comment slightly: Technical federation is NOT a path forward for dataspaces. However, dataspace might create Federations at the level of Business Processes or as Legal Entities. This should be clearly distinguished.

**Comment by ssteinbuss** [comment]
This text is outdated and the relevant content is part of the new section on [Interoperability](https://github.com/International-Data-Spaces-Association/IDSA-Rulebook/blob/main/documentation/3_Interoperability.md). This PR should be rejected and closed.

**Comment by PeterKoen-MSFT** [comment]
@ssteinbuss I noticed this PR being open for a long time and the comments not being resolved... shall we have another go at trying to resolve the comments together to finally merge the changes?

**Comment by PeterKoen-MSFT** [comment]
rejected as per @ssteinbuss comments.

**Pull request closed without merging**